src/index.html

```diff
@@ -1,13 +1,68 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+<html lang="en" class="card">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="body card__content">
+
+    <div class="body__body" data-qa="card">
+
+      <div class="body__item-picture">
+        <img class="body__imac-image"
+        src="./images/imac.jpeg"
+        alt="imac">
+      </div>
+
+      <section class="body__item-name">
```

Comment:
no need to use `section` here
it's not a  thematic grouping of content
https://www.w3schools.com/html/html5_semantic_elements.asp

Reply:
also, make card  as a separate `card` bem block, not a `body`